phpstan-baseline.neon

```diff
@@ -0,0 +1,20 @@
+parameters:
+    ignoreErrors:
+        -
+            # TODO: Remove this integration, the last version of this dependency is too old to be installed
```

Comment:
Should we create an issue about this ?

Reply:
You got too fast, read my other comment.